Mathlib/MeasureTheory/Measure/MeasureSpace.lean

```diff
@@ -1004,6 +1004,9 @@ theorem lt_iff' : μ < ν ↔ μ ≤ ν ∧ ∃ s, μ s < ν s :=
 instance instAddLeftMono {_ : MeasurableSpace α} : AddLeftMono (Measure α) :=
   ⟨fun _ν _μ₁ _μ₂ hμ s => add_le_add_left (hμ s) _⟩
```

Comment:
Do we want to keep this here, or we can remover it and let it be inferred from `IsOrderedAddMonoid`?